identity/src/main/java/com/android/identity/PresentationHelper.java

```diff
@@ -108,7 +107,7 @@ public class PresentationHelper {
      */
     public PresentationHelper(@NonNull Context context,
                               @NonNull PresentationSession presentationSession,
-                              @NonNull DataTransport transport,
+                              @Nullable DataTransport transport, //TODO: make it NonNull
```

Comment:
Please revert this file, it is not required to have nullable transport.

Reply:
Sure!